.github/workflows/workflow.yml

```diff
@@ -24,8 +24,11 @@ jobs:
       - uses: actions/checkout@v3
       - uses: actions/setup-node@v3
         with:
-          node-version: "12"
+          node-version: "16"
+          cache: "npm"
+          cache-dependency-path: setup/package-lock.json
       - run: npm ci --prefer-offline --no-audit --progress=false
+      - run: npm run bundle
```

Comment:
Would be nice to add this step here as well, but this doesn't work on windows and adding another job for this might be overkill.. wdyt?

What it does is check if the above `npm run bundle` generated a different dist/index.js which means the committer forgot to run `npm run bundle` :)

```yaml
      - name: Check dist/index.js was generated
        run: |
          git diff --text dist/index.js
          [[ -z $(git status -s) ]]
```

Reply:
Ooh, I like it.

```yaml
      - name: Check dist/index.js was generated
        if: "matrix.os == 'ubuntu-latest'"
        run: |
          git diff --text dist/index.js
          [[ -z $(git status -s) ]]
```

would work to only run the check on ubuntu without requiring a whole separate job